src/StatusQ/Buttons/StatusIconButton.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+
+Rectangle {
+    id: statusIconButton
+
+    property color iconNormalColor: "transparent"
+    property color iconDisabledColor: "transparent"
+
+    property color backgroundNormalColor: "transparent"
+    property color backgroundHoverColor: "transparent"
+    property color backgroundDisabledColor: "transparent"
+    property alias iconName: icon.icon
+
+    property bool showBackground: false
+
+    property bool loading: false
+
+    signal pressed(var mouse)
+
+
+
+    /// Implementation
+
+    implicitWidth: 44// by design
+    implicitHeight: 44// by design
+    radius: showBackground ? width / 2
+                           : 8
+
+    color: {
+        if (!showBackground) {
+            return sensor.containsMouse ? backgroundHoverColor
+                                        : "transparent"
+        }
+
+        if (statusIconButton.enabled) {
+            return sensor.containsMouse ? backgroundHoverColor
+                                        : backgroundNormalColor
+        } else {
+            return backgroundDisabledColor
+        }
+    }
+    MouseArea {
+        id: sensor
+
+        anchors.fill: parent
+        cursorShape: Qt.PointingHandCursor
+        hoverEnabled: !loading
+
+
+        StatusIcon {
+            id: icon
+            anchors.centerIn: parent
+            visible: !loading
+
+            width: statusIconButton.width -  20
+            height: statusIconButton.height - 20
+
+            color: {
+                if (statusIconButton.enabled) {
+                    return iconNormalColor
+                } else {
+                    return iconDisabledColor
+                }
+            }
+        } // Icon
+        StatusLoadingIndicator {
+            anchors.centerIn: parent
+            visible: loading
+            color: {
+                if (statusIconButton.enabled) {
+                    return iconNormalColor
+                } else {
+                    return iconDisabledColor
+                }
+            }
+        } // Indicator
```

Comment:
Similar to `StatusButton`: should this be rather rendered using a `Loader`?

Reply:
I think it must be `Loader`, thanks. But maybe we could fix indictor like this:
`
StatusIcon {
    id: statusIcon
    icon: "loading"
    height: 17
    width: 17
    RotationAnimator {
        target: statusIcon;
        from: 0;
        to: 360;
        duration: 1200
        running: statusIcon.visible
        loops: Animation.Infinite
    }
}
`

Reply:
So mean, we shouldn't use `StatusLoadingIndicator` and instead conditionially apply the animator?

Because that's eactly what `StatusLoadingIndicator` does.

Reply:
No we just take care about "always running animation". If you check `BusyIndicator` in QML you can find public property for stop/start it.

Reply:
Okay, happy to see that in a PR :)